rascal-lsp/src/main/rascal/demo/lang/pico/LanguageServer.rsc

```diff
@@ -36,17 +36,22 @@ module demo::lang::pico::LanguageServer
 import util::LanguageServer;
 import util::IDEServices;
 import ParseTree;
+import util::ParseErrorRecovery;
 import util::Reflective;
 import lang::pico::\syntax::Main;
 
+private Tree picoParser(str input, loc origin) {
+    return parse(#start[Program], input, origin, allowRecovery=true, filters={createParseErrorFilter(false)});
```

Comment:
is there no way to use the `parser` function to build a function that does this?